test/e2e/operator_test.go

```diff
@@ -2139,6 +2139,290 @@ func TestContainerLogging(t *testing.T) {
+	route := &routev1.Route{}
+	if err := kclient.Get(context.TODO(), routeName, route); err != nil {
+		t.Fatalf("failed to get the console route: %v", err)
+	}
+	clientPod := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "headertest",
+			Namespace: podList.Items[0].Namespace,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:    "curl",
+					Image:   podList.Items[0].Spec.Containers[0].Image,
+					Command: []string{"/bin/curl"},
+					Args: []string{
+						"-k",
+						"-o", "/dev/null", "-s",
+						"--resolve",
+						route.Spec.Host + ":443:" + podList.Items[0].Status.PodIP,
+						"https://" + route.Spec.Host,
+					},
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyNever,
+		},
+	}
+	if err := kclient.Create(context.TODO(), clientPod); err != nil {
+		t.Fatalf("failed to create pod %s/%s: %v", clientPod.Namespace, clientPod.Name, err)
+	}
+	defer func() {
+		if err := kclient.Delete(context.TODO(), clientPod); err != nil {
+			if apierrors.IsNotFound(err) {
+				return
+			}
+			t.Fatalf("failed to delete pod %s/%s: %v", clientPod.Namespace, clientPod.Name, err)
+		}
+	}()
+
+	kubeConfig, err := config.GetConfig()
+	if err != nil {
+		t.Fatalf("failed to get kube config: %v", err)
+	}
+	client, err := kubernetes.NewForConfig(kubeConfig)
+	if err != nil {
+		t.Fatalf("failed to create kube client: %v", err)
+	}
+
+	err = wait.PollImmediate(1*time.Second, 3*time.Minute, func() (bool, error) {
+		for _, pod := range podList.Items {
+			readCloser, err := client.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, &corev1.PodLogOptions{
+				Container: "logs",
+				Follow:    false,
+			}).Stream(context.TODO())
+			if err != nil {
+				t.Errorf("failed to read logs from pod %s: %v", pod.Name, err)
+				continue
+			}
+			data, err := ioutil.ReadAll(readCloser)
+			if err != nil {
+				t.Errorf("failed to read logs from pod %s: %v", pod.Name, err)
+				continue
+			}
+			scanner := bufio.NewScanner(bytes.NewBuffer(data))
+			var found bool
+			for scanner.Scan() {
+				var length int
+				line := scanner.Text()
+				length = len(line)
+				// rsyslog generate a little overhead in the logline, the overhead is based on the name of the POD where the router is running
+				// the POD name can be different for each test and how the name is build can be changed in the future, so it is not possible to
+				// set a fixed value for the test
+				if strings.Contains(line, "8192abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@@@@@@@=") && (length >= 8100 && length <= 8400) {
+					t.Logf("found log message for maxLength and the length is equals to 8192 chars in pod %s logs: %s", pod.Name, line)
+					found = true
+					break
+				}
+			}
+			if err := readCloser.Close(); err != nil {
+				t.Errorf("failed to close logs reader for pod %s: %v", pod.Name, err)
+			}
+			if !found {
+				t.Logf("failed to find output:\n\n%s", string(data))
```

Comment:
nit: Since the data can be very long, it is better not to print it out as it will fill up the test logs.
```suggestion
				t.Logf("failed to find log message for maxLength test")
```

Reply:
done